europython2024/README.md

```diff
@@ -0,0 +1,14 @@
+# No more spaghetti code: build reusable data pipeline with Sciline
+
+## Abstract
+
+Have you dealt with scary spaghetti code that was not written by yourself?
```

Comment:
I'm also not so sure `sciline` can help someone that has to make sense of someone else's old code?
It would if the old code was written with `sciline`, but it most probably wouldn't be?

Reply:
We will work on the abstract together tomorrow, no more need to comment here ;)

Reply:
It's also good to collect opinions here in advance : D....